xla/python/ops.cc

```diff
@@ -371,7 +371,11 @@ void BuildOpsSubmodule(nb::module_& m) {
   ops.def("Clamp", &Clamp, nb::arg("min"), nb::arg("operand"), nb::arg("max"));
   ops.def("Collapse", &Collapse, nb::arg("operand"), nb::arg("dimensions"));
   ops.def("CollectivePermute", &CollectivePermute, nb::arg("operand"),
-          nb::arg("source_target_pairs"), nb::arg("channel_id") = std::nullopt);
+          nb::arg("source_target_pairs"), nb::arg("channel_id") = std::nullopt,
+          nb::arg("inplace") = false);
+  ops.def("MultiCollectivePermute", &MultiCollectivePermute,
+          nb::arg("operands"), nb::arg("source_target_pairs"),
+          nb::arg("channel_id") = std::nullopt, nb::arg("inplace") = false);
```

Comment:
Could you please update their corresponding definitions in `//xla/python/xla_extension/ops.pyi` as well?
E.g.,
https://github.com/openxla/xla/blob/0f6331b1881ae34c8b1cd59580900d556bc8305c/xla/python/xla_extension/ops.pyi#L135-L138

Reply:
updated